modules/library/src/it/scala/zio/elasticsearch/HttpExecutorSpec.scala

```diff
@@ -377,6 +377,81 @@ object HttpExecutorSpec extends IntegrationSpec {
             ElasticExecutor.execute(ElasticRequest.deleteIndex(firstSearchIndex)).orDie
           )
         ) @@ shrinks(0),
+        suite("searching documents using PIT and returning them as a stream")(
+          test("successfully create point in time and return stream results") {
+            checkOnce(genCustomer) { customer =>
+              def sink: Sink[Throwable, Item, Nothing, Chunk[Item]] =
+                ZSink.collectAll[Item]
+
+              for {
+                _ <- ElasticExecutor.execute(ElasticRequest.deleteByQuery(secondSearchIndex, matchAll))
+                reqs = (0 to 200).map { _ =>
+                         ElasticRequest.create[CustomerDocument](
+                           secondSearchIndex,
+                           customer.copy(id = Random.alphanumeric.take(5).mkString, balance = 150)
+                         )
+                       }
+                _    <- ElasticExecutor.execute(ElasticRequest.bulk(reqs: _*).refreshTrue)
+                query = range("balance").gte(100)
+                res <- ElasticExecutor
+                         .largeStream(ElasticRequest.search(secondSearchIndex, query))
+                         .run(sink)
+              } yield assert(res)(hasSize(equalTo(201)))
+            }
+          } @@ around(
+            ElasticExecutor.execute(ElasticRequest.createIndex(secondSearchIndex)),
+            ElasticExecutor.execute(ElasticRequest.deleteIndex(secondSearchIndex)).orDie
+          ),
+          test("successfully create point in time and return stream results as specific Type") {
```

Comment:
Should we use `PIT` here as well? Also, I would write `type` instead of `Type`.

Reply:
I can use both, PIT(point in time).